### ISSUE TITLE
"Local (GPU)" environment information is not adapting the resize:A11y_ML.NET Model Builder_Environment_Resize

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3032]
### GitHubTags
#A11ySev3; #A11yTCS;#Model Builder - Machine Learning for .NET Developers;#DesktopApp; #Win32;#BM-MlnetModelBuilder-Win32-March25; #Win32;#WCAG1.4.4;#Resize;#A11yMAS; 

**Prerequisites to apply resize:**

Method1:
Settings -> Change the display resolution to 1366X768 and drag the text size slider to 200%.
Method2:
Settings -> Change the display resolution to 1366X768 and scale to 200% and sign out.

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Prerequisites:
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment (Select Azure).
6. Verify that "Local (GPU)" environment information is not adapting the resize 

### Actual Result:
 "Local (GPU)" environment information is not adapting the resize with the 200% text resize method and with the scaling 200% content is not visible.

### Expected Result:
On Applying Resize "Local (GPU)" environment information should adapt, and information should be clearly visible.
### User Impact:
Low vision users who zoom the page for better visibility won't be able to see the information properly if it doesn't adapt to the resize.

### Attachment: 

Before Resize

![Image](https://github.com/user-attachments/assets/5db64219-1171-465f-86ba-3b51742b4141)

Resize with text 200.

![Image](https://github.com/user-attachments/assets/dd704c70-987c-4ad8-926b-82b805ab3e38)

Resize with scaling 200%

![Image](https://github.com/user-attachments/assets/e585f63f-a764-4dc2-973e-9b400ce95c68)
